From: Aaron Simmons <paleozogt@gmail.com>
Date: Sat, 15 Feb 2020 12:42:01 -0800
Subject: Disabling compilation of type_map.go for gccgo.

Origin: backport, https://github.com/modern-go/reflect2/commit/ce5cf29be307ab6039519b733b1861dde5f61c40
---
 type_map.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/type_map.go b/type_map.go
index 3acfb55..38abcbf 100644
--- a/type_map.go
+++ b/type_map.go
@@ -1,3 +1,5 @@
+// +build !gccgo
+
 package reflect2
 
 import (
